### DISCUSSION TITLE
if users can select the font for text in default editor？

### DISCUSSION THREAD

**Discussion by iOSyan** [hm21/pro_image_editor#89] [Q&A] [answered]
### Your question

Awesome!
May I ask if users can select the font for text in default editor ?

**Comment by hm21** [comment, marked as answer]
Yes, I just released version `2.6.8` where it is also possible to choose between different fonts in the default editor. Just set your settings in the configurations like below:

```dart
textEditorConfigs: TextEditorConfigs(
    customTextStyles: [
      // List of your text styles
      GoogleFonts.roboto(),
      GoogleFonts.averiaLibre(),
      GoogleFonts.lato(),
      GoogleFonts.comicNeue(),
      GoogleFonts.actor(),
      GoogleFonts.odorMeanChey(),
      GoogleFonts.nabla(),
    ],
  ),
``` 
You can see a complete example [here](https://github.com/hm21/pro_image_editor/blob/71e1bd258d6ae54b0bbe1ecba51206407895e37b/example/lib/pages/google_font_example.dart#L35).
